LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -93,10 +99,22 @@ struct FHIRResourcesView: View {
             EmptyView()
         }
     }
-    
+
+    private var filteredResourceKeys: [String] {
+        resources.keys.sorted().filter { resourceType in
+            if let resourceArray = resources[resourceType] {
+                return searchText.isEmpty || resourceArray.contains { resource in
+                    resource.displayName.lowercased().contains(searchText.lowercased())
+                }
+            }
+            return false
+        }
+    }
     
     private func resources(for resourceType: String) -> some View {
-        ForEach(resources[resourceType] ?? []) { resource in
+        ForEach((resources[resourceType] ?? []).filter { resource in
+            searchText.isEmpty || resource.displayName.lowercased().contains(searchText.lowercased())
+        }) { resource in
```

Comment:
I would suggest to move this computation into a private computed property in the view to make the syntax a bit easier and less nested 👍